exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,21 @@
 
 let studentCount = 16;
 let mentorCount = 9;
-let moreStudentsThanMentors; // finish this statement
-
+if (studentCount > mentorCount) {
+return moreStudentsThanMentors = true; // finish this statement
```

Comment:
you'd want to assign the variable, not return the result of assigning a variable (which will be `undefined`) e.g. 
```
if (studentCount > mentorCount) {
   moreStudentsThanMentors = true
}
```

or

```
const moreStudentsThanMentors = studentCount > mentorCount
```